### DISCUSSION TITLE
A friendlier third-person perspective

### DISCUSSION THREAD

**Discussion by PSheon** [unavi-xyz/unavi#697] [Ideas]
Update the perspective to facilitate more interaction with the scene in the future.
like this.

<img width="423" alt="截圖 2023-03-23 下午2 01 35" src="https://user-images.githubusercontent.com/20603727/227117076-ac907489-34b7-4a39-936a-9a825471dde7.png">

**Comment by PSheon** [comment]
@kayhhh ^^

**Comment by m11m** [comment]
So move the camera a bit to the side, so you can see where you are clicking. Might make rotating around the character feel a bit weird which I don't love, but I think you're right that it would be better for interaction. I think its worth trying out, shouldn't be too hard to implement. Can always add a setting too to use centered vs offset third person mode.
